src/components/AttachedActivity.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useMemo } from 'react';
+import { userOrDefault } from '../utils';
+
+import { EnrichedActivity, OGAPIResponse } from 'getstream';
+
+import { Thumbnail } from 'react-file-utils';
+
+export type FileData = {
+  mimeType: string;
+  name: string;
+  url: string;
+};
+
+export type Attachments = {
+  files?: FileData[];
+  images?: string[];
+  og?: OGAPIResponse;
+};
+
+export type CustomActivityData = {
+  attachments?: Attachments;
+  image?: string;
+  link?: boolean;
+  text?: string;
+};
+
+export type UserData = {
```

Comment:
```suggestion
type UserData = {
```

Reply:
these types have a generics name and they will clash with other types in the project since we export * the types